build/BUILD

```diff
@@ -160,3 +171,31 @@ filegroup(
         "//test/kubemark:all-srcs",
     ],
 )
+
+# These bundles group images at no build time cost unless a .tar is built from
+# the bundle. We use these bundles to push all images from the pushall rule.
+docker_bundle(
+    name = "all_images",
+    images = {
+        "gcr.io/google_containers/%s:{STABLE_DOCKER_TAG}" % (binary): binary + "-internal"
```

Comment:
do we need to depend on the `-internal` one here? or would the result of `docker_bundle` work? I guess it probably doesn't matter?